.github/autolabeler.yml

```diff
@@ -0,0 +1,24 @@
+'addon: a11y': ["addons/a11y/**"]
+'addon: actions': ["addons/actions/**"]
+'addon: backgrounds': ["addons/backgrounds/**"]
+'addon: centered': ["addons/centered/**"]
+'addon: events ': ["addons/events/**"]
+'addon: graphql ': ["addons/graphql/**"]
+'addon: info': ["addons/info/**"]
+'addon: jest': ["addons/jest/**"]
+'addon: knobs': ["addons/knobs/**"]
+'addon: links': ["addons/links/**"]
+'addon: notes': ["addons/notes/**"]
+'addon: options': ["addons/options/**"]
+'addon: storyshots': ["addons/storyshots/**"]
+'addon: viewport': ["addons/viewport/**"]
+'app: angular': ["app/angular/**"]
+'app: polymer ': ["app/polymer/**"]
+'app: react-native': ["app/react-native/**"]
+'app: react': ["app/react/**"]
+'app: vue': ["app/vue/**"]
+'babel / webpack': ["webpack", "babel"]
+'cli': ["lib/cli/**"]
+'compatibility with other tools': []
+'documentation': ["docs", "*.md"]
```

Comment:
Would you consider adding things to any of `example/*` as a documentation change?  Or changing things in `core` / linter files / as a maintenance?

Reply:
Shall we merge and add details later?